### PR TITLE
fix(TFD-2647): Use default String as output type converter.

### PR DESCRIPTION
**Pull request by RyanSkraba**
**What is the current behavior?** (You can also link to an open issue here)

The default value for the output type of a TypeConverter is empty (unspecified).

**What is the new behavior?**

The default value for the output type of a TypeConverter is String.

**Please check if the PR fulfills these requirements**

- [X] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [X] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/1422/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/1422/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Arskraba%2Ffix%2FTFD-2647-typeconverter-default)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/1423/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/1423/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Arskraba%2Ffix%2FTFD-2647-typeconverter-default)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/1424/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/1424/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Arskraba%2Ffix%2FTFD-2647-typeconverter-default)

File: components/components-processing/processing-definition/src/main/java/org/talend/components/processing/definition/typeconverter/TypeConverterProperties.java [hunk 1]
```diff
@@ -84,8 +83,8 @@ public Converter getConverter(){
 
         public Property<String> field = PropertyFactory.newString("field").setRequired().setValue("");
 
-        public Property<TypeConverterOutputTypes> outputType = PropertyFactory
-                .newEnum("outputType", TypeConverterOutputTypes.class).setRequired();
+        public Property<String> outputType = PropertyFactory.newString("outputType", "String", "String")
```
Comment (lbourgeois) [inline components/components-processing/processing-definition/src/main/java/org/talend/components/processing/definition/typeconverter/TypeConverterProperties.java]: I would have found more elegant to stay with EnumProperty but I am ok with this if it is easier

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/1435/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/1435/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Arskraba%2Ffix%2FTFD-2647-typeconverter-default)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/1436/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/1436/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Arskraba%2Ffix%2FTFD-2647-typeconverter-default)

**Pull request merged**